src/page-modules/contact/layouts/ticket-control-page-layout.tsx

```diff
@@ -35,7 +35,7 @@ function ComplaintPageLayout({ children }: ComplaintPageLayoutProps) {
             checked={router.pathname.includes('/utsette-betaling')}
             onChange={() =>
               router.push(
-                '/contact/billettkontroll-og-gebyr/klage',
+                '/contact/billettkontroll-og-gebyr/utsette-betaling',
```

Comment:
You don't have to do anything about it in this PR, but we should probably use `kontakt` instead of `contact` when the rest of the URL is in Norwegian

Reply:
Agreed👍